aspnetcore/mvc/views/tag-helpers/built-in/image-tag-helper.md

```diff
@@ -1,52 +1,53 @@
 ---
 title: Image Tag Helper in ASP.NET Core
 author: pkellner
-description: Shows how to work with Image Tag Helper
+description: Shows how to work with Image Tag Helper.
 ms.author: riande
-ms.date: 02/14/2017
+ms.custom: mvc
+ms.date: 10/10/2018
 uid: mvc/views/tag-helpers/builtin-th/image-tag-helper
 ---
 # Image Tag Helper in ASP.NET Core
 
-By [Peter Kellner](http://peterkellner.net) 
+By [Peter Kellner](http://peterkellner.net)
 
-The Image Tag Helper enhances the `img` (`<img>`) tag. It requires a `src` tag as well as the `boolean` attribute `asp-append-version`.
+The Image Tag Helper enhances the `<img>` tag to provide cache-busting behavior for static image files.
 
-If the image source (`src`) is a static file on the host web server, a unique cache busting string is appended as a query parameter to the image source. This ensures that if the file on the host web server changes, a unique request URL is generated that includes the updated request parameter. The cache busting string is a unique value representing the hash of the static image file.
+For an overview of Tag Helpers, see <xref:mvc/views/tag-helpers/intro>.
 
-If the image source (`src`) isn't a static file (for example a remote URL or the file doesn't exist on the server), the `<img>` tag's `src` attribute is generated with no cache busting query string parameter.
+The Image Tag Helper requires a `src` attribute and the boolean attribute `asp-append-version`.
+
+If the image source (`src`) is a static file on the host web server, a unique cache busting string is appended as a query parameter to the image source. If the file on the host web server changes, a unique request URL is generated that includes the updated request parameter. The cache busting string is a unique value representing the hash of the static image file.
```

Comment:
If the image source (`src`) is a static file on the host web server:

*  a unique cache busting string is appended as a query parameter to the image source. 
* If the file on the host web server changes, a unique request URL is generated that includes the updated request parameter. The cache busting string is a unique value representing the hash of the static image file.

Try to keep sentence max to 15-18 words.